deployments/managed-services.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: "Managed services"
+---
+
+One of the key features of Architect.io is its dependency graph, which is a visual representation of the relationships between different services in an application.
```

Comment:
I think we can remove "visual" here. It's more of a concept than just something that we display